README.md

```diff
@@ -13,10 +13,11 @@ npm start
 
```

Comment:
Please update it all from top to bottom.

Reply:
Update completed! Thx